From 939daba04c25a9a3b7be2f3df39692453cf9a304 Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Thu, 21 Mar 2013 14:57:18 +0000
Subject: Missing dependency of librelogo on writer

...at least due to dependency of librelogo.xcd on writer.xcd, see
82c53d537a05dadf4d7fd7ea41292897bf2d47c7 "Missing dependency."

Otherwise, having librelogo installed but not writer will cause an uncaught
RuntimeException from configmgr::Components::parseXcdFiles
(configmgr/source/components.cxx) early on in soffice.bin.

(cherry picked from commit 4e69b35674882d8d675a677940805b187cad0bc3)
Conflicts:
	setup_native/source/packinfo/packinfo_office.txt

Change-Id: I97565fe5c790ed182bb27fd722c650acf8a8ee08
Reviewed-on: https://gerrit.libreoffice.org/2898
Reviewed-by: Fridrich Strba <fridrich@documentfoundation.org>
Reviewed-by: Andras Timar <atimar@suse.com>
Reviewed-by: Petr Mladek <pmladek@suse.cz>
Tested-by: Petr Mladek <pmladek@suse.cz>
---
diff --git a/setup_native/source/packinfo/packinfo_office.txt b/setup_native/source/packinfo/packinfo_office.txt
index 3e8c89e..cbf8c490 100644
--- a/setup_native/source/packinfo/packinfo_office.txt
+++ b/setup_native/source/packinfo/packinfo_office.txt
@@ -314,10 +314,10 @@ End
 Start
 module = "gid_Module_Optional_Pyuno_LibreLogo"
 solarispackagename = "%BASISPACKAGEPREFIX%WITHOUTDOTPRODUCTVERSION-librelogo"
-solarisrequires = "%BASISPACKAGEPREFIX%WITHOUTDOTPRODUCTVERSION-core01, SUNWPython"
+solarisrequires = "%BASISPACKAGEPREFIX%WITHOUTDOTPRODUCTVERSION-core01,%BASISPACKAGEPREFIX%WITHOUTDOTPRODUCTVERSION-writer,SUNWPython"
 packagename = "%BASISPACKAGEPREFIX%PRODUCTVERSION-librelogo"
-freebsdrequires = "%BASISPACKAGEPREFIX%PRODUCTVERSION-core01"
-requires = "%BASISPACKAGEPREFIX%PRODUCTVERSION-core01"
+freebsdrequires = "%BASISPACKAGEPREFIX%PRODUCTVERSION-core01,%BASISPACKAGEPREFIX%PRODUCTVERSION-writer"
+requires = "%BASISPACKAGEPREFIX%PRODUCTVERSION-core01,%BASISPACKAGEPREFIX%PRODUCTVERSION-writer"
 copyright = "2012 The Document Foundation"
 solariscopyright = "solariscopyrightfile"
 vendor = "The Document Foundation"
--
cgit v0.9.0.2-2-gbebe
